src/codemodder/cli.py

```diff
@@ -44,7 +44,7 @@ class CsvListAction(argparse.Action):
     """
 
     def __call__(self, parser, namespace, values, option_string=None):
-        items = set(values.split(","))
+        items = values.split(",")
```

Comment:
Order matters now. But we might want to validate that the IDs are unique some other way.